### ISSUE TITLE
Could not get any generated file when I run the command

### ISSUE THREAD

**Issue by saidkacem** [FrancescoCatani/svg_to_paint_dart#2]
Hello,

When I run the following command : 
- flutter pub run svg_to_paint C:\Users\HOME\Downloads\diamond.xml C:\Users\HOME\Downloads\ icon absolute

I get :
Modules installed...
Files Generated...
Files Created...

But no file is created in the given directory.

Could you tell more how to proceed ? I may miss something

Regards

**Comment by FrancescoCatani** [comment]
Hello, i will write you all process to generate your file.

1. Add the package in yaml file flutter pub add svg_to_paint
2. Activate it! flutter pub global activate svg_to_paint
3. Important! Get your file.svg that you want to draw and copy its on this site https://svg2vector.com (use this site to be sure). The site will allows you to download a zip file which will contain a file.xml (this is the file that you have to use with package.)
4. Once have you done that you can type svg_to_paint source_path/file.xml destination_path file_name absolute

Example: 
`svg_to_paint /Users/myuser/Desktop/svg_to_paint/file.xml /Users/myuser/Desktop/svg_to_paint/ icon absolute`

will generate the follow:

<img width="638" alt="example" src="https://user-images.githubusercontent.com/56845074/117509838-3c2f9b80-af8b-11eb-80bb-396f0dfcee0a.png">

Be sure you have converted the svg file in a right way! Let me know. Waiting for news...

Cheers

**Comment by saidkacem** [comment]
Hello,

To be sure, we are on the same page, the command line "svg_to_paint ....." is not runnable by itself. I should add "flutter pub" as prefix. right ?
Cause when I run " svg_to_paint /Users/myuser/Desktop/svg_to_paint/file.xml /Users/myuser/Desktop/svg_to_paint/ icon absolute" I get the following error :
- svg_to_paint : The term 'svg_to_paint' is not recognized as the name of a cmdlet ....

I am working on windows. Is there any compatibility issue ?

thank you for your feedback.

**Comment by 0xfc10** [comment]
You have to set you environmental variables on Windows.Follow this guide https://dart.dev/tools/pub/cmd/pub-global (Running a script from your PATH). Do it and let me know

**Comment by FrancescoCatani** [comment]
You have to set you environmental variables on Windows.Follow this guide https://dart.dev/tools/pub/cmd/pub-global (Running a script from your PATH). Do it and let me know

**Comment by saidkacem** [comment]
Hello again, 

I did set the env variable to include the %LOCALAPPDATA%\Pub\Cache\bin.  When I launch the command, I still get the same error which make sense because the package must be declared as executable and it is not the case. I could not find it in "..\Pub\Cache\bin". see picture attached.

![Capture](https://user-images.githubusercontent.com/78408152/117519502-16fb5700-afa4-11eb-9653-a31b6cab7a7d.PNG)

When I run 
*dart pub global run svg_to_paint C:\Users\HOME\AndroidStudioProjects\svg_to_paint_dart\images\car.xml .\AndroidStudioProjects\svg_to_paint_dart\images\  icon123435434543 absolute

Iget this message "svg_to_paint 1.0.9 requires the Flutter SDK, which is unsupported for global executables."

The only way to get something running is to execute the command :
* flutter pub run svg_to_paint C:\Users\HOME\AndroidStudioProjects\svg_to_paint_dart\images\car.xml C:\Users\HOME\AndroidStudioProjects\svg_to_paint_dart\images\  icon123435434543 absolute

In this case I get 
Modules installed...
Files Generated...
Files Created...

But no file is created. 

I commented  line 54 in your bin/svg_to_paint.dart file 
....
        print("Files Created...");
        //Directory("dir").deleteSync(recursive: true);   <-- this line

in order to keep the 'dir' created. When I look inside index.js I see the following: 
const svgToPaint = require('svg_to_paint')
svgToPaint.generateDartCode('C:\Users\HOME\AndroidStudioProjects\svg_to_paint_dart\images\car.xml','C:\Users\HOME\AndroidStudioProjects\svg_to_paint_dart\images\','icon123435434543','absolute')

It seems to be good but I don't understand why  it does not generate the good files after this step.

Regards

**Comment by FrancescoCatani** [comment]
You have to find the right path of cache in your disk before add it on environmental variables. Anyway, do this thing, clone the repository, with your prompt go in /bin project folder (where is svg_to_paint.dart) and type: 
`dart run svg_to_paint.dart ...` followed of arguments that you need. It works! There is a possibility that your file has something wrong, let me know if it works, in other case, if you can you could send me the .svg file and i will test it to check.

**Comment by saidkacem** [comment]
I did what you suggested, same result. 
![Capture](https://user-images.githubusercontent.com/78408152/117538944-64a9ab00-b008-11eb-8fbc-0cb425af4a7e.PNG)

the file I used is a simple one : 
[atom.zip](https://github.com/FrancescoCatani/svg_to_paint_dart/files/6445833/atom.zip)

By the way, I also commented line 54 in "svg_to_paint_dart" so I can see what was generated in "index.js" before its execution via node.
![Capture 2](https://user-images.githubusercontent.com/78408152/117539048-fe715800-b008-11eb-9332-9447b90bd6de.PNG)
 
The created index.js file is here : 
![Capture 3](https://user-images.githubusercontent.com/78408152/117539347-80ae4c00-b00a-11eb-8c24-cee3ebac2448.PNG)

Any futher taughts ?

Thank you  for helping !

**Comment by FrancescoCatani** [comment]
Ok, I tried. With this specific formatted file the package doesn't work...I will investigate more in the next days to check if there is a problem with code. Have you used https://svg2vector.com/ to create it? If the problem is the file there is an high probability that when the program parses the xml file it breakes with a specific tag but it's strange because I tested its with a lot of svgs and it works.

**Comment by saidkacem** [comment]
Yes, I have created it using https://svg2vector.com/. the svg origin file is : https://dev.w3.org/SVG/tools/svgweb/samples/svg-files/atom.svg 

ok, keep in touch
Thanks

**Comment by FrancescoCatani** [comment]
Hello sorry for the wait but I've been busy and first of all I want to thank you for your report. I investigated, the problem is with parsing the svg with the 2 groups (where is 'android:rotation'). Actually the lib isn't able to parse this specific type of file to dart code and I should check some things to implement it well. If you need to generate automatic dart code from this specific svg file I suggest you to delete the 2 groups tag and then when you will have generate the painter you can add the 2 sub groups manually.

**Comment by yk81708090** [comment]
I have the same problem. How can I solve it？

**Comment by yk81708090** [comment]
its music score svg file

**Comment by prateekmedia** [comment]
@FrancescoCatani Same behaviour with following close.xml file:
```xml
<vector xmlns:android="http://schemas.android.com/apk/res/android" xmlns:aapt="http://schemas.android.com/aapt"
    android:viewportWidth="16"
    android:viewportHeight="16"
    android:width="16dp"
    android:height="16dp">
    <group
        android:translateX="-60"
        android:translateY="-518">
        <group
            android:translateX="19"
            android:translateY="-242">
            <path
                android:pathData="M45 764l1 0c0.01037 -0.00012 0.02079 -0.00046 0.03125 0 0.254951 0.0112 0.50987 0.12858 0.6875 0.3125L49 766.59375 51.3125 764.3125C51.578125 764.082 51.759172 764.007 52 764l1 0 0 1c0 0.28647 -0.03434 0.55065 -0.25 0.75l-2.28125 2.28125 2.25 2.25C52.906938 770.46942 52.999992 770.7347 53 771l0 1 -1 0c-0.265301 -0.00001 -0.530586 -0.0931 -0.71875 -0.28125L49 769.4375 46.71875 771.71875C46.530586 771.90694 46.26529 772 46 772l-1 0 0 -1c-0.000003 -0.26529 0.09306 -0.53058 0.28125 -0.71875l2.28125 -2.25L45.28125 765.75C45.070508 765.55537 44.97809 765.28075 45 765l0 -1z"
                android:fillColor="#BEBEBE" />
        </group>
    </group>
</vector>
```
